sites/public/pages/eligibility/age.tsx

```diff
@@ -49,6 +56,12 @@ const EligibilityAge = () => {
         />
       </FormCard>
       <FormCard>
+        <FormBackLink
+          url={eligibilityRoute(CURRENT_PAGE - 1)}
+          onClick={() => {
+            // Not extra actions needed.
+          }}
```

Comment:
Do we need the empty `onClick()` for this to work? If not, can we remove it?

Reply:
It's required in the component unfortunately.